config/custom-environment-variables.yml

```diff
@@ -38,3 +38,9 @@ public:
     sessionReplaySampleRate: 'DATADOG_SESSION_REPLAY_SAMPLE_RATE'
     sessionSampleRate: 'DATADOG_SESSION_SAMPLE_RATE'
     site: 'DATADOG_SITE'
+  default_otp:
+    enabled: 'DEFAULT_OTP_ENABLED'
+    code: 'DEFAULT_OTP_CODE'
+    # Whitelist is handled as a comma-separated string in environment variables
+    # and will be parsed by the application
+    whitelist: 'DEFAULT_OTP_WHITELIST'
```

Comment:
This is a poor variable name, and does not convey that its list of phone numbers.

Rename to `whitelisted_phone_numbers_with_country_code` (or similar) so the one setting does not have to look at the code base on how it is supposed to be used.